services/catarse/app/models/balance_transaction.rb

```diff
@@ -64,23 +65,25 @@ def refresh_metadata
     Raven.extra_context({})
   end
 
-  def self.insert_balance_transfer_between_users(from_user, to_user)
+  def self.insert_balance_transfer_between_users(from_user, to_user, amount_to_transfer = nil)
     from_user.reload
     return if from_user.total_balance <= 0
+    return if from_user.total_balance < amount_to_transfer && amount_to_transfer.present?
```

Comment:
O `amount_to_transfer.present?` tem que vir primeiro, pois se for `nil` ali na comparação vai dar erro.